packages/backend/src/lightdash.ts

```diff
@@ -1,27 +1,40 @@
-import {
-    attachTypesToModels,
-    convertExplores,
-    getDbtCatalog,
-    getDbtModels
-} from "./dbt/translator";
-import {
-    isDbtProcessRunning,
-    refreshDbtChildProcess, spawnDbt
-} from "./dbt/childProcess";
-import {MissingCatalogEntryError, NotExistsError} from "./errors";
+import {NotExistsError} from "./errors";
 import {Explore, MetricQuery} from "common";
-import {cache} from "./cache";
-import {runQueryOnDbtAdapter, waitForDbtServerReady} from "./dbt/rpcClient";
 import {buildQuery} from "./queryBuilder";
+import {DbtLocalProjectAdapter} from "./projectAdapters/dbtLocalProjectAdapter";
+import {DbtRemoteProjectAdapter} from "./projectAdapters/dbtRemoteProjectAdapter";
+import {ProjectAdapter} from "./types";
 
+// TODO: WIP FOR REFACTOR OF CONFIG FILES
+// Setup dbt adapter
+let adapter: ProjectAdapter;
+const spawnDbt = process.env.LIGHTDASH_SPAWN_DBT === 'false' ? false : true;
```

Comment:
Making code more reliable:
- Check for `true` value and consider everything else as `false`. Example: with the previous code, if they misspelled 'flase' it would spawn dbt
- Remove unnecessary inline condition 

```suggestion
const spawnDbt = process.env.LIGHTDASH_SPAWN_DBT === 'true';
```